### PR TITLE
Feat/empty and sample filled schemas

### PR DESCRIPTION
**Pull request by DoubleCouponDay**
**Objective:** To update the database repository with new schemas that support an end to end test environment on a local machine. Docker should be able to load an empty database or the samples database using commands that a developer or research can understand and use quickly.

**Changes:**

- Renamed cases schema to main.

- Renamed pipeline_cases schema to main_preparsed.

- A new ingest schema that supports the data-aggregator.

- Moved schemas to their own folder.

- run.sh script now performs based on the variable 'empty' or 'samples'.
 
- docker-compose.yml is split into two files, depending on the use case.

- Readme.md has been updated appropriately.

- Moved samples into separate schemas that are readable in a text editor and still zipped in consideration of repository limitations.

### CODE REVIEW

**Comment by DoubleCouponDay** [comment]
Closed until new sample data has been generated.

**Comment by DoubleCouponDay** [comment]
As requested, no sample data has been generated for the new ingest schema.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+/schemas/filled/
```
Comment (mcqueen1987) [inline .gitignore]: There should be a new line at the end of each file.

File: README.md [hunk 2]
```diff
@@ -4,21 +4,49 @@ Use this repository to get up and running with the OpenLaw NZ database for local
...
+
+- With test data: 
+```
+    docker-compose --file docker-compose.samples.yml up --build --renew-anon-volumes --detach
+```
 
-    docker-compose up [-d]
+- Just empty tables:
+```
+    docker-compose --file docker-compose.empty.yml up --build --renew-anon-volumes --detach
```
Comment (mcqueen1987) [inline README.md]: I have deploied and runned this command line in my local environment. it works well. 
image

File: init-scripts/run.sh [hunk 3]
```diff
@@ -1,4 +1,54 @@
...
+sqllabel='.sql'
+mainlabel='main'
+ingestlabel='ingest'
+preparsedlabel='main_preparsed'
+filledlabel='filled_'
+restoringlabel='restoring'
+emptylabel='empty'
+sampleslabel='samples'
+name='run.sh'
+postgreslabel='postgres'
```
Comment (mcqueen1987) [inline init-scripts/run.sh]: The naming of each parameters should be camelCase (postgresLabel) or UnderScoreCase (postgres_label) which is easier to read.

File: .gitignore (hunk 1 above)
Comment (DoubleCouponDay) [reply .gitignore]: Fixed.

File: README.md (hunk 2 above)
Comment (DoubleCouponDay) [reply README.md]: Should be "deployed".

File: init-scripts/run.sh (hunk 3 above)
Comment (DoubleCouponDay) [reply init-scripts/run.sh]: I think style conventions are out of scope but we can discuss it with William on Tuesday.

**Pull request merged**